_posts/2023-04-04-segment-replication.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title:  "Reduce compute costs and increase throughput with segment replication, available GA in OpenSearch 2.7."
+authors:
+- satnandi
+date: 2023-04-04
+categories:
+ - technical-post
+meta_keywords: 
+meta_description: 
+
+excerpt: We are excited to announce that segment replication---a new replication strategy introduced in OpenSearch 2.3---is generally available in version 2.7. Segment replication is an alternative to document replication, where documents are copied from primary shards to their replicas. With segment replication, index segments are transferred instead, so the heavy-duty indexing workload is done on the primary shard, significantly increasing index throughput and lowering compute costs. In this blog post, we will dive deep into the concept of segment replication, its advantages and shortcomings, and planned future enhancements.
```

Comment:
I think this need some re-wording. you need to focus on that on DocRep, replica need to perform the same indexing operation that happened on primary. On the other side, on SegRep we only copy the segment files instead